apps/speed.c

```diff
@@ -1444,12 +1444,6 @@ int speed_main(int argc, char **argv)
             continue;
         }
 #ifndef OPENSSL_NO_RSA
-# ifndef RSA_NULL
-        if (strcmp(*argv, "openssl") == 0) {
```

Comment:
Removing this means that anyone using this on the command line will get "unknown algorithm". Probably better to keep this, but just not call RSA_set_default_method()